### DISCUSSION TITLE
Need help integrating Bulma template in NextJs

### DISCUSSION THREAD

**Discussion by AsiwajuDev** [vercel/next.js#15665] [Help]
I have a template built fully functional with Bulma and I'm trying to convert the whole template into NextJs still using Bulma.

**Comment by stigkj** [comment]
Is a Bulma template HTML+CSS? Then just overwrite Next.js' main template by creating your own _document.jsx and put the Bulma template into that.

**Reply by AsiwajuDev** [reply]
It's a Bulma template of HTML and CSS. Already used _document.js to override by linking the assets from Public folder. The assets is loaded into the resources but when I copy anything from the template, I get a blank page with nothing rendering
